src/PrestaShopBundle/Form/Admin/Sell/Product/ShortcutsType.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product;
+
+use PrestaShopBundle\Form\Admin\Sell\Product\Shortcut\PriceShortcutType;
+use PrestaShopBundle\Form\Admin\Sell\Product\Shortcut\StockShortcutType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ShortcutsType extends TranslatorAwareType
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('stock', StockShortcutType::class, [
+                'label' => $this->trans('Quantity', 'Admin.Catalog.Feature'),
+                'help' => $this->trans('How many products should be available for sale?', 'Admin.Catalog.Help'),
+                'target_tab' => 'stock-tab',
+                'target_tab_name' => $this->trans('Quantity', 'Admin.Catalog.Feature'),
+            ])
+            ->add('price', PriceShortcutType::class, [
+                'label' => $this->trans('Price', 'Admin.Global'),
```

Comment:
However, here maybe we could write 'Retail price' to match the label on the 'Pricing section' and the word employed in the help text. Wdyt?

Reply:
Oh this one I need to change in any case I think, it doesn't match the current naming used in Shortcuts anyway
![Capture d’écran 2021-03-15 à 11 22 42](https://user-images.githubusercontent.com/13801017/111138956-d6bdc080-8580-11eb-986e-d7687c60169f.png)

Let me know which one should be used, should we use stricly the same as in the Tab (with "Retail price") or a shorter one since it's in a shortcut section

Reply:
Following your comment about the confusion between 'Retail price' and 'Cost price', I think we should use 'Retail price' everywhere for more consistency, even though it's a shortcut section.